content/docs/deployment/helm/modules/installation/observability/deployment.md

```diff
@@ -180,12 +162,15 @@ The Container Storage Modules (CSM) for Observability Helm chart bootstraps an O
        kubectl get configmap [POWERMAX-CONFIG-PARAMS] -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/name: [POWERMAX-CONFIG-PARAMS]/name: powermax-config-params/' | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
        ```
 
-    4. Copy the `karavi-authorization-config`, `proxy-server-root-certificate`, `proxy-authz-tokens` Secret from the CSI PowerMax namespace into the CSM for Observability namespace:
+    3. Copy the `karavi-authorization-config`, `proxy-server-root-certificate`, and `proxy-authz-tokens` Secrets from the CSI PowerMax namespace into the CSM for Observability namespace:
 
        ```bash
 
        kubectl get secret karavi-authorization-config proxy-server-root-certificate proxy-authz-tokens -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | sed 's/name: karavi-authorization-config/name: powermax-karavi-authorization-config/' | sed 's/name: proxy-server-root-certificate/name: powermax-proxy-server-root-certificate/' | sed 's/name: proxy-authz-tokens/name: powermax-proxy-authz-tokens/' | kubectl create -f -
        ```
+       **Note:** The config map approach is being phased out. However, for backwards compatibility, you can still configure and use the Observability module with PowerMax driver using the config map.
```

Comment:
same as my other comment on using deprecated instead of phased out.

Reply:
Done. Made the change at all the places to maintain consistency across the documentation.